package.json

```diff
@@ -47,7 +47,7 @@
     "@loopback/context": "^1.0.0",
     "@loopback/core": "^1.0.0",
     "@loopback/rest": "^1.1.0",
-    "express-jwt": "^5.3.1",
+    "express-jwt": "github:KalleV/express-jwt#dad0daebe354a9ad0fc8ab160e406fd5a3cac7d9",
```

Comment:
Can we avoid dependencies on your private repo?

Reply:
@keatsk I wish we could, but express-jwt refuses to merge my PR: https://github.com/auth0/express-jwt/pull/203. They're using an outdated function detection check.

Reply:
Can we move the fork into Labshare?

Reply:
Sure, I don't have permission to do so though